lib/samson/secrets/vault_server.rb

```diff
@@ -73,13 +73,12 @@ def sync!(other)
 
       def create_client
         VaultClientWrapper.new(
-          DEFAULT_CLIENT_OPTIONS.merge(
+          **DEFAULT_CLIENT_OPTIONS,
             address: address,
             ssl_cert_store: cert_store,
             ssl_verify: tls_verify,
             token: token,
             versioned_kv: versioned_kv?
```

Comment:
these should be on the same level
```suggestion
          address: address,
          ssl_cert_store: cert_store,
          ssl_verify: tls_verify,
          token: token,
          versioned_kv: versioned_kv?
```